analyzer/codechecker_analyzer/ctu_manager.py

```diff
@@ -112,7 +112,7 @@ def generate_ast(triple_arch, action, source, config, env):
 
     cmdstr = ' '.join(cmd)
     LOG.debug_analyzer("Generating AST using '%s'", cmdstr)
```

Comment:
```suggestion
    LOG.debug_analyzer("Generating AST using '%s'", ' '.join(cmd))
```
so we would not need `cmdstr`?

Reply:
This variable is used a few lines lower in an error message print.